tests/runtimes/android/android_app_gradle_tests.py

```diff
@@ -0,0 +1,235 @@
+        assert ':asbg:generateBindings', 'Static Binding Generator not executed'
+        assert 'cannot access its superclass' not in output.output
+
+    def test_420_include_gradle_flavor(self):
+        # https://github.com/NativeScript/android-runtime/pull/937
+        # https://github.com/NativeScript/nativescript-cli/pull/3467
+        Folder.clean(os.path.join(TEST_RUN_HOME, APP_NAME))
+        Tns.create(app_name=APP_NAME, template=Template.HELLO_WORLD_JS.local_package, update=True)
+        source = os.path.join(TEST_RUN_HOME, 'assets', 'runtime', 'android', 'files', 'android-runtime-pr-937',
+                              'app.gradle')
+        target = os.path.join(TEST_RUN_HOME, APP_NAME, 'app', 'App_Resources', 'Android', 'app.gradle')
+        File.copy(src=source, target=target)
+
+        Tns.platform_add_android(APP_NAME, framework_path=Android.FRAMEWORK_PATH)
+
+        Tns.build_android(os.path.join(TEST_RUN_HOME, APP_NAME), verify=False)
+
+        assert File.exists(os.path.join(TEST_RUN_HOME, APP_NAME,
+                                        "platforms", "android", "app", "build", "outputs", "apk", "arm64Demo", "debug",
+                                        "app-arm64-demo-debug.apk"))
+        assert File.exists(os.path.join(TEST_RUN_HOME, APP_NAME,
+                                        "platforms", "android", "app", "build", "outputs", "apk", "arm64Full", "debug",
+                                        "app-arm64-full-debug.apk"))
+        assert File.exists(os.path.join(TEST_RUN_HOME, APP_NAME,
+                                        "platforms", "android", "app", "build", "outputs", "apk", "armDemo", "debug",
+                                        "app-arm-demo-debug.apk"))
+        assert File.exists(os.path.join(TEST_RUN_HOME, APP_NAME,
+                                        "platforms", "android", "app", "build", "outputs", "apk", "armFull", "debug",
+                                        "app-arm-full-debug.apk"))
+        assert File.exists(os.path.join(TEST_RUN_HOME, APP_NAME,
+                                        "platforms", "android", "app", "build", "outputs", "apk", "x86Demo", "debug",
+                                        "app-x86-demo-debug.apk"))
+        assert File.exists(os.path.join(TEST_RUN_HOME, APP_NAME,
+                                        "platforms", "android", "app", "build", "outputs", "apk", "x86Full", "debug",
+                                        "app-x86-full-debug.apk"))
+
+    def test_440_verify_no_class_not_found_exception_is_thrown(self):
+        """
+        ClassNotFound exception when calling nested static class with correct argument
+        https://github.com/NativeScript/android-runtime/issues/1195
+        """
+
+        Folder.clean(os.path.join(TEST_RUN_HOME, APP_NAME))
+        Tns.create(app_name=APP_NAME, template=Template.HELLO_WORLD_JS.local_package, update=True)
```

Comment:
Can we use Template.MIN_JS?

Reply:
All tests are based on hello world js template. I would prefer using it.

Reply:
Why? Do you test something specific to webpack?